### PR TITLE
ci-operator/config/openshift/release: Add credentials-request-freeze for 4.7 and 4.8

### PR DESCRIPTION
**Pull request by wking**
Extend 9e91c0d3dc (#24126) to our older, supported z streams, because we don't want surprise cred request bumps in those either.

~For all three, I've picked the oldest supported 4.y.z:~ edit: no longer testing 4.6, and 4.7 and 4.8 have been raised past the GCP `skipServiceCheck` addition.

```console
$ for V in 4.6 4.7 4.8; do curl -s "https://api.openshift.com/api/upgrades_info/v1/graph?channel=stable-${V}" | jq -r ".nodes[].version | select(startswith(\"${V}.\"))" | sort -V | head -n1; done
4.6.1
4.7.0
4.8.2
```

### CODE REVIEW

**Comment by wking** [comment]
[4.7][2] and [4.8][3] both failed on `skipServiceCheck`, so bumped to 4.8.21 and 4.7.38.

[Green for 4.6][1], which is a bit strange.  Inspecting the 4.6 history:

```console
$ curl -s "https://api.openshift.com/api/upgrades_info/v1/graph?channel=fast-4.6" | jq -r '.nodes[].version | select(startswith("4.6."))' | sort -V | while read V; do oc adm release extract --credentials-requests --to "${V}" "quay.io/openshift-release-dev/ocp-release:${V}-x86_64"; done
$ A=
$ ls -d 4.6.* | sort -V | while read B; do if test -n "${A}"; then diff -ru "${A}" "${B}"; fi; A="${B}"; done
Only in 4.6.13: 0000_50_cloud-credential-operator_05-s3-credentialsrequest.yaml
$ oc adm release extract --credentials-requests --to 4.6.0-0.nightly-2021-12-01-092838 registry.ci.openshift.org/ocp/release:4.6.0-0.nightly-2021-12-01-092838
$ diff -ru 4.6.49/0000_30_machine-api-operator_00_credentials-request.yaml 4.6.0-0.nightly-2021-12-01-092838/0000_30_machine-api-operator_00_credentials-request.yaml
--- 4.6.49/0000_30_machine-api-operator_00_credentials-request.yaml     2021-12-01 12:32:24.969630046 -0800
+++ 4.6.0-0.nightly-2021-12-01-092838/0000_30_machine-api-operator_00_credentials-request.yaml  2021-12-01 13:23:38.166465406 -0800
@@ -106,6 +106,7 @@
     - roles/compute.instanceAdmin.v1
     - roles/iam.serviceAccountUser
     - roles/compute.loadBalancerAdmin
+    skipServiceCheck: true
   secretRef:
     name: gcp-cloud-credentials
     namespace: openshift-machine-api
```

And then all in one big hop:

```console
$ diff -ru 4.6.1 4.6.0-0.nightly-2021-12-01-092838
diff -ru 4.6.1/0000_30_machine-api-operator_00_credentials-request.yaml 4.6.0-0.nightly-2021-12-01-092838/0000_30_machine-api-operator_00_credentials-request.yaml
--- 4.6.1/0000_30_machine-api-operator_00_credentials-request.yaml      2021-12-01 12:21:54.332663831 -0800
+++ 4.6.0-0.nightly-2021-12-01-092838/0000_30_machine-api-operator_00_credentials-request.yaml  2021-12-01 13:23:38.166465406 -0800
@@ -106,6 +106,7 @@
     - roles/compute.instanceAdmin.v1
     - roles/iam.serviceAccountUser
     - roles/compute.loadBalancerAdmin
+    skipServiceCheck: true
   secretRef:
     name: gcp-cloud-credentials
     namespace: openshift-machine-api
Only in 4.6.1: 0000_50_cloud-credential-operator_05-s3-credentialsrequest.yaml
```

So... I'm not sure why the CI job didn't pick that up...

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/24166/rehearse-24166-periodic-ci-openshift-release-master-nightly-4.6-credentials-request-freeze/1466134330056118272
[2]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/24166/rehearse-24166-periodic-ci-openshift-release-master-nightly-4.7-credentials-request-freeze/1466134329267589120
[3]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/24166/rehearse-24166-periodic-ci-openshift-release-master-nightly-4.8-credentials-request-freeze/1466134328370008064

**Comment by wking** [comment]
While I sort out [the 4.6.1 vs. 4.6-nightly issue][1]

/hold

[1]: https://github.com/openshift/release/pull/24166#issuecomment-984068893

**Comment by wking** [comment]
Ah, the issue with 4.6 is that 4.6's `oc adm release extract --credentials-requests --to ...` seems to ignore `--to` and just print to stdout.  I'm picking openshift/oc#482 back to 4.6 with openshift/oc#984.  Until that lands, I'm dropping the 4.6 job.

**Comment by wking** [comment]
[4.7][1] and [4.8][2] are both green, as is [4.9][3] (so my step command changes didn't break that).  I think we're good to go here.

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/24166/rehearse-24166-periodic-ci-openshift-release-master-nightly-4.7-credentials-request-freeze/1466167421822832640
[2]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/24166/rehearse-24166-periodic-ci-openshift-release-master-nightly-4.8-credentials-request-freeze/1466167420992360448
[3]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/24166/rehearse-24166-periodic-ci-openshift-release-master-nightly-4.9-credentials-request-freeze/1466167422653304832

**Review by vrutkovs — APPROVED** [review]
/lgtm

**Pull request merged**
